buildshiprun/handler.go

```diff
@@ -215,6 +215,14 @@ func Handle(req []byte) string {
 		userAnnotations := buildAnnotations(annotationWhitelist, event.Annotations)
 		userAnnotations[sdk.FunctionLabelPrefix+"git-repo-url"] = event.RepoURL
 
+		if val, ok := os.LookupEnv("fn_write_timeout"); ok {
```

Comment:
Could you rather create a function for this, that returns a value or the default?

Then take the return value and populate it in the event's environment after that?

It will also enable you to write a unit test in this PR before merging.

Reply:
@alexellis Fixed